apps/labs/posts/close-encounter-with-pandas-and-the-jedis-of-open-source.md

```diff
@@ -11,184 +11,189 @@ hero:
-who was doing more reading and communicating rather than writing code. But luckily "*being a developer is 
-more than just typing code*", paraphrasing Malissa's comment when I made a pseudo-complaint about my 
-internship experience thus far (bearly a month in at the time).
-
-Since most people working on open-source projects are usually not paid to make contributions, there are 
-lots of features to be implemented, tests to be performed and documentation to write. Noa mentioned that 
-part of being a maintainer was doing actual maintenance and this involves things like "testing all pandas 
-operations with empty inputs and documenting the findings", "improving the documentation for some 
-operation or method" or "writing an internal documentation for a feature".
-
-There was a need to understand what was expected and having read the already cold discussion on the issue 
-to maintain consistency for panda's operation on empty inputs, It turned out though that there haven't 
-been many tests written on empty inputs and some of the results are a little unpredictable. For example, given
+Having been involved in other internships, with the most recent being the [MLH
+Fellowship](https://fellowship.mlh.io) and
+[Outreachy](https://www.outreachy.org/), I had to write a significant amount of
+code which was my metric for measuring a developer at the time. This idea nursed
+a little bit of doubt in me as an intern who was doing more reading and
+communicating rather than writing code. But luckily "_being a developer is more
+than just typing code_", paraphrasing Melissa's comment when I made a
+pseudo-complaint about my internship experience thus far (barely a month in at
+the time).
+
+Since most people working on open-source projects are usually not paid to make
+contributions, there are lots of features to be implemented, tests to be
+performed and documentation to write. Noa mentioned that part of being a
+maintainer was doing actual maintenance and this involves things like "testing
+all pandas operations with empty inputs and documenting the findings",
+"improving the documentation for some operation or method" or "writing an
+internal documentation for a feature".
+
+There was a need to understand what was expected and having read the already
+cold discussion on the issue to maintain consistency for panda's operation on
+empty inputs, It turned out though that there haven't been many tests written on
+empty inputs and some of the results are a little unpredictable. For example,
+given
 
 ```python
     import pandas as pd
 
     empty_df = pd.DataFrame({"a": [], "b": []})
     gb = empty_df.groupby("a", group_keys=True)
     group = getattr(gb, "b")
-    group.apply("sum") # Returns an empty series, which is understandable since the group is empty.
+    group.apply("sum")
+    # Returns an empty series, which is understandable since the group is empty.
```

Comment:
Newlines to make the code easier to read on smaller, narrower screens

Reply:
Looks good. 👍